gb-ppu/src/ppu.rs

```diff
@@ -40,6 +44,20 @@ impl PPU {
         &mut self.control
     }
 
+    pub fn bg_palette_mut(&mut self) -> &mut Palette {
+        &mut self.bg_palette
+    }
+    pub fn obj_palette_0_mut(&mut self) -> &mut Palette {
+        &mut self.obj_palette.0
+    }
+    pub fn obj_palette_1_mut(&mut self) -> &mut Palette {
+        &mut self.obj_palette.1
+    }
+
+    // pub fn control_mut(&mut self) -> &mut Control {
+    //     &mut self.control
+    // }
```

Comment:
to remove if not used.
if this function will be required in the future: extract that method into its own branch

Reply:
this was a duplicate code.